content/cli/plugins.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: bt_wiki
+title: plugins
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+weight: 160
+---
+
+The `cfy plugins` command is used to manage plugins stored on a Cloudify manager.
+
+You can use the command to upload, download, delete and list plugins and also to get information on a specific plugin.
+
+A Cloudify plugin is an archive created by [wagon]({{< relref "http://github.com/cloudify-cosmo/wagon" >}}).
+
+See [plugins]({{< relref "plugins/overview.md" >}}) for more information.
+
+
+## Commands
+
+### upload
+
+Usage: `cfy plugins upload -p PLUGIN_FILE`
+
+Upload a plugin to the manager.
+
+#### Required flags
+
+* `-p, --plugin-path=PLUGIN_FILE` - The path to the Cloudify plugin (`.wgn` file) you would like to upload. Note that only `tar.gz` formatted wagons are allowed.
```

Comment:
The term `tar.gz` formatted wagons is somewhat confusing. I know what you mean here, but a user may be confused as to whether wgn is expected? tar.gz? etc...